.github/workflows/integration.yml

```diff
@@ -12,7 +12,7 @@ jobs:
           submodules: 'recursive'
       - uses: actions/setup-node@v2
         with:
-          node-version: '14'
```

Comment:
If `16.16.0` is the requirement, we need to update the README.md as well as is still states Node 14 (see Line 31).

Reply:
Good point, I will update it in the feat/minimal-web-ui branch since we will merge that one first.